packages/components/src/components/table/table-row/table-row-styles.ts

```diff
@@ -6,17 +6,30 @@ import {
   hostHiddenStyles,
   hoverMediaQuery,
 } from '../../../styles';
+import { borderRadiusSmall } from '@porsche-design-system/utilities-v2';
+import type { Theme } from '../../../types';
 
-export const getComponentCss = (): string => {
+export const offset = '-2px';
```

Comment:
This makes it overlap into the item above and after?

Reply:
yes

Reply:
replaced it with `getInsetJssStyle`